packages/utils/src/calls.rs

```diff
@@ -0,0 +1,185 @@
+use serde::{de::DeserializeOwned, Serialize};
+
+use cosmwasm_std::{
+    to_binary, Coin, CosmosMsg, HumanAddr, Querier, QueryRequest, StdResult, Uint128, WasmMsg,
+    WasmQuery,
+};
+
+use super::space_pad;
+
+/// A data structure representing the instantiation message of a contract
+///
+/// This trait provides a method to create the CosmosMsg used to instantiate a contract
+pub trait InitCallback: Serialize {
```

Comment:
Saying "A data structure" here is misleading since it isn't a data structure.
I'd phrase it as something like "A trait marking types that can be used to instantiate contracts".
The second line is good, but i would add:
"This trait **requires specifying a padding block size,** and provides a method to create the CosmosMsg used to instantiate a contract"

Reply:
@reuvenpo I looked at https://docs.serde.rs/serde/ser/trait.Serialize.html and https://docs.serde.rs/serde/trait.Deserialize.html as an example for docstrings on a trait and thought maybe it was conventional to start it like that, describing what the data structure should be if you intend to implement the trait

Reply:
hmm, that's interesting.
I never noticed that approach before. I think that there isn't really a convention for this kind of stuff.
If you think this is a better way to explain it, then we can keep it this way :)

Reply:
I think it is better to change it.  Just the fact that you mentioned this means that how it is worded does not make it clear that it is trying to qualify what the data structure should be if it is going to implement the trait